src/_guides/platforms.md

```diff
@@ -25,16 +25,16 @@ or server, there's a Dart solution for that.
 
   Write an app that runs in any modern web browser.
 
-- {:.card.narrow}[![Dart logo][]{: width="100"}][Dart VM]{:.no-automatic-external}
+- {:.card.narrow}[![Dart logo][]{: width="100"}][Server]{:.no-automatic-external}
```

Comment:
The second sentence of the opening paragraph further above reads:

> Whether you're creating a mobile app, web app, command-line script, or server, there's a Dart solution for that.

Questions:

- Do we still want to mention "command-line", or should it be removed so as to match the platform list?
- How about replacing "server" by "server-side app"?